semantickernel-experimental/src/main/java/com/microsoft/semantickernel/data/vectorstorage/options/GetRecordOptions.java

```diff
@@ -7,10 +7,26 @@
  * Options for getting a record.
  */
 public class GetRecordOptions {
+
     private final boolean includeVectors;
 
-    public GetRecordOptions(boolean includeVectors) {
+    private final boolean wildcardKeyMatching;
+
```

Comment:
May mentioning that wildCarKeyMatching is only available for SQL related vector collectios.